sympy/series/fourier.py

```diff
@@ -154,9 +154,29 @@ def _eval_subs(self, old, new):
             return self
 
     def truncate(self, n=3):
-        """Returns the first n (non-zero)terms of the series.
+        """
+        Return the first n nonzero terms of the series.
+
+        If n is None return an iterator.
+
+        :param n: amount of non-zero terms in approximation
+        :type n: int or None
+        :rtype: Expr or iterator
+        :return: sigma approximation of function expanded into Fourier series
```

Comment:
Do you mean something like docstring below, without using Sphinx-like parameters documentation, just plain description?

```
def truncate(self, n=3):
    """
    Return the first n nonzero terms of the Fourier series expantion.

    If n is None return an iterator.

    Examples
    ========

    >>> from sympy import fourier_series, pi
    >>> from sympy.abc import x
    >>> s = fourier_series(x, (x, -pi, pi))
    >>> s.truncate(4)
    2*sin(x) - sin(2*x) + 2*sin(3*x)/3 - sin(4*x)/2

    See Also
    ========

    sympy.series.fourier.FourierSeries.sigma_approximation
    """
    ...
```